### ISSUE TITLE
Ability to use permutations and combinations in evaluate()

### ISSUE THREAD

**Issue by mathletedev** [josdejong/mathjs#2314]
Just a small feature request: Is it possible to have permutations/combinations in the evaluate() function? Something like `5 choose 3` or `5C3` would be nice. Thanks!

**Comment by vels10** [comment]
Can i able to solve this Issue?

**Comment by josdejong** [comment]
Yeah, so you would like to introduce two new operators to enter permutations/combinations in a more compact way. That would be nice indeed I'm not sure this is a broad enough use case to introduce two new operators though, but we can think this through at least.

What operator names would you suggest for both functions? It should be really clear from the names whether this is about permutations or combinations.

**Comment by m93a** [comment]
On calculators, combination and permutation numbers are typically written as `nCr` and `nPr`. What about adding these two functions as shorthands for `combinations` and `permutations`?
```javascript
math.nCr(5,3) // 10
math.nPr(4,2) // 12
```

**Comment by josdejong** [comment]
Yes that is right indeed, `nCr` and `nPr`. I would like to keep the amount of "duplicate" functions minimal though, try to have only one way to do things. To prevent people from wondering "is `nCr` the same as `combinations` or is it doing different things?". 

Maybe we could introduce a solution that gives you an error message with a hint when you enter "nCr": "You used 'nCr' you mean 'combinations'?", having defined a list with aliases and maybe also common typos in function names. Or maybe it's just ok like it is now and we should not over-engineer this 😁 .

**Comment by gwhitney** [comment]
Since this seems to be an open-ended idea (possibly add a facility for "function suggestions"), re-titling and moving to Discussions.
